dashboard/models.py

```diff
@@ -261,10 +280,10 @@ def get_course_resource_type(self, course_id):
 
 
 class ResourceManager(models.Manager):
-    def get_queryset(self):
+    def get_queryset(self) -> QueryType:
         return ResourceQuerySet(self.model, using=self._db)
 
-    def get_course_resource_type(self, course_id):
+    def get_course_resource_type(self, course_id: models.BigIntegerField) -> Optional[List[str]]:
```

Comment:
Seeing a couple instances of `models.BigIntegerField` here and below in type hints. Were those you checked were satisfied with?